### PR TITLE
allow searching by multiple organizations (separated by commas)

### PR DESCRIPTION
**Pull request by drewda**
when listing repos for more than one organization, each repo is identified by its full name

### CODE REVIEW

**Comment by drewda** [comment]
@mourner: thanks to you and Mapbox for creating and sharing this. I'm working with repos across many orgs, so I just added the ability to aggregate results across multiple orgs. Not sure if it's also useful to you, but here's a PR in case you'd like it.

**Comment by mourner** [comment]
That's awesome! I'll gladly merge it of course. :)

File: index.js [hunk 1]
```diff
@@ -83,8 +88,9 @@ function addRepos(repos) {
 
     for (var i = 0; i < repos.length; i++) {
         var repo = repos[i];
+        var repo_name = numOrgs > 1 ? repo.full_name : repo.name
```
Comment (mourner) [inline index.js]: camelCase please

**Comment by drewda** [comment]
`can_you_tell_i_use_ruby?`

**Pull request merged**